### DISCUSSION TITLE
Extra Row and Inner Column

### DISCUSSION THREAD

**Discussion by dieselmat** [joshreve/dactyl-keyboard#56] [Q&A]
Hello,

Thanks for the amazing work. I'm just wondering if its possible to general the same bottom row and inner column like the one below:
<img width="1191" alt="Screenshot 2021-10-29 at 9 44 53 PM" src="https://user-images.githubusercontent.com/15997914/139447942-7b252a5a-f520-4f6a-8acb-6a2d67e1dd51.png">

**Comment by joshreve** [comment]
I have it half in the works.  It got temporarily sidelined for other efforts but is definitely possible.  I'll take a look as you are the second person to directly request it.  The only complexity is the number of variations and how many places the column key counts have to be accounted for.

**Reply by dieselmat** [reply]
Cool! Maybe if you can share it in a branch, I can try to test and/or contribute to the code.
